core/model/modx/modfilehandler.class.php

```diff
@@ -431,6 +431,37 @@ public function save($content = null, $mode = 'w+') {
         return $result;
     }
 
+     /**
+     * Writes a CSV file.
+     *
+     * @param array $content An array that holds one csv line per key, each column is a sub-array item
+     * @param string $mode The mode in which to write
+     * @param array $options Optional options for fputcsv, here you can specify custom delimiter and enclosure
+     * @return boolean true when the operation was successful
+     */
+    public function saveCSV($content = array(), $mode = 'w+', $options = array()) {
+        $options = array_merge(array(
+            'delimiter' => ',',
+            'enclosure' => '"',
+        ), $options);
+
+        $result = false;
+
+        if (!empty($content)) {
+            $fp = @fopen($this->path, $mode);
```

Comment:
haha, that wasn't me...have a look at all the other "original" modFileHandler methods, they are full of @'s for error suppression, I just used the normal save() method as a template when I wrote this...